db/migrate/20170922212510_add_severity_to_miq_alerts.rb

```diff
@@ -0,0 +1,5 @@
+class AddSeverityToMiqAlerts < ActiveRecord::Migration[5.0]
```

Comment:
This needs to be added to manageiq-schema

Reply:
Wow, thanks @bdunne. My first migration after the split. I'll open a PR there.